src/components/UpcomingEventsList.tsx

```diff
@@ -109,35 +104,15 @@ const UpcomingEventsList = () => {
   }
 
   return (
-    <>
+    <div className="">
       {Object.keys(monthGroupedEvents)
         ?.slice(0, maxRange)
         ?.map((month) => {
           const events = monthGroupedEvents[month]
           return (
-            <Box key={`events_in_${month}`}>
-              <Heading
-                as="h3"
-                fontSize={{ base: "xl", md: "2xl" }}
-                py={8}
-                fontWeight="semibold"
-                lineHeight={1.4}
-              >
-                {month}
-              </Heading>
-              <Grid
-                width={{ base: "100%" }}
-                templateColumns={{
-                  base: "repeat(1,1fr)",
-                  md: "repeat(2,1fr)",
-                  lg: "repeat(3,1fr)",
-                }}
-                justifyContent={"center"}
-                alignItems={"center"}
-                alignSelf={"center"}
-                gap={5}
-                overflow={"hidden"}
-              >
+            <div key={`events_in_${month}`}>
+              <h3 className="py-8 text-2xl font-semibold">{month}</h3>
```

Comment:
Based on your comment, I agree, there are a lot of old styles that do not match the DS.

Since you are adapting the line height, we could also just use the standard `h3` size that matches the DS styles.

```suggestion
              <h3 className="py-8">{month}</h3>
```